### PR TITLE
ft: ZENKO-1718 MongoClientInterface putObj tests

### PR DESCRIPTION
**Pull request by philipyoo**
some functional tests for MongoClientInterface putObject cases

setup provided by @bennettbuchanan  :) ty

### CODE REVIEW

File: tests/functional/metadata/utils/fakeLogger.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const fakeLogger = {
```
Comment (bennettbuchanan) [inline tests/functional/metadata/utils/fakeLogger.js]: This can be imported from `tests/utils/fakeLogger`.

**Review by bennettbuchanan — APPROVED** [review]
LGTM!

**Review by dora-korpar — APPROVED** [review]
lgtm!

**Comment by philipyoo** [comment]
@bert-e approve

**Pull request merged**